openhcl/virt_mshv_vtl/src/processor/tdx/mod.rs

```diff
@@ -843,6 +843,14 @@ impl BackingPrivate for TdxBacked {
 
             state.commit().expect("committing state should succeed");
         }
+
+        // TODO TDX: TDX shares the vp context page for xmm registers only. It
```

Comment:
This TODO is out of date.

Reply:
What should it be? Are we happy with the current state or do we still want to move things around?

Reply:
I don't think we'll want to move it to it's own page.